pkg/controller/incompatible_operator_controller.go

```diff
@@ -68,7 +68,7 @@ func (c *incompatibleOperatorController) sync(ctx context.Context, _ factory.Syn
 	var updateStatusFn v1helpers.UpdateStatusFunc
 	incompatibleOperators, err := c.getIncompatibleOperators()
 	if len(incompatibleOperators) > 0 {
-		message := fmt.Sprintf("Found ClusterExtensions that require upgrades prior to upgrading cluster to version %d.%d: %s.", c.nextOCPMinorVersion.Major, c.nextOCPMinorVersion.Minor, strings.Join(incompatibleOperators, ","))
+		message := fmt.Sprintf("Found ClusterExtensions that require upgrades prior to upgrading cluster to version %d.%d: %s.", c.currentOCPMinorVersion.Major, c.currentOCPMinorVersion.Minor, strings.Join(incompatibleOperators, ","))
```

Comment:
If the cluster is 4.19.0-xxxx, the error message is like bellow:
InstalledOLMOperatorsUpgradeable: Found ClusterExtensions that require upgrades prior to upgrading cluster to version 4.19: bundle xxxxx}
should it be "Found ClusterExtensions that require upgrades prior to upgrading cluster to version 4.20"?